dll/win32/advapi32/token/token.c

```diff
@@ -100,7 +100,7 @@ CheckTokenMembership(IN HANDLE ExistingTokenHandle,
                                          0,
                                          sizeof(SECURITY_DESCRIPTOR) +
                                              sizeof(ACL) + SidLen +
-                                             sizeof(ACCESS_ALLOWED_ACE));
+                                             sizeof(ACCESS_ALLOWED_ACE) - sizeof(ULONG));
```

Comment:
Would `sizeof (ACCESS_ALLOWED_ACE.SidStart)` work, as in linked documentation?

Reply:
No, it's not a valid syntax. I've seen usage of [`sizeof(ULONG)`](https://github.com/reactos/reactos/blob/6c75215bb3473b984b8667a9589b39dcc19eade0/drivers/filesystems/btrfs/security.c#L381), [`sizeof(DWORD)`](https://github.com/reactos/reactos/blob/e90316aeaa4c844b0f589a349bd8178b06eaeea2/base/system/winlogon/wlx.c#L980) and [`sizeof(UINT32)`](https://github.com/microsoft/Windows-driver-samples/blob/6c1981b8504329521343ad00f32daa847fa6083a/network/trans/WFPSampler/exe/HelperFunctions_CommandLine.cpp#L118). But `SidStart` is defined as `ULONG` in the struct, so I believe using `ULONG` is the way to go.

Reply:
Let's use an NT-"compliant" syntax instead:
```
RTL_FIELD_SIZE(ACCESS_ALLOWED_ACE, SidStart)
```

Reply:
Even better: `FIELD_OFFSET(ACCESS_ALLOWED_ACE, SidStart)` gives the size with the last member already subtracted.

Reply:
@ThFabba [it's being used, too](https://github.com/reactos/reactos/blob/e90316aeaa4c844b0f589a349bd8178b06eaeea2/base/system/winlogon/wlx.c#L1052). Changed my pull request to use `FIELD_OFFSET`.